crates/levm/src/operations.rs

```diff
@@ -187,3 +211,84 @@ impl Operation {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
For now, let's move these tests to `crates/levm/operations_tests.rs`. But we'll probably remove them in the future since this functionality is only meant for testing and we're not entirely sure that EVM's users will use this, in addition to this, having these cases covered in the `VM`'s tests makes these unit tests needless.

For future EVM opcodes implementations, skip adding unit tests for `Operation`.

Reply:
Got it!